HW4.py

```diff
@@ -0,0 +1,16 @@
+# readcsv.py -path_csv example.csv -col_name Marker_Strategy
+
+import argparse
+import csv
+import sys
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-path_csv')
+parser.add_argument('-col_name')
+ns = parser.parse_args(sys.argv[1:])
+
+result = []
+with open(ns.path_csv) as f:
+    reader = csv.DictReader(f)  # read rows into a dictionary format
+    result = [row[ns.col_name.replace('_', ' ')] for row in reader]
```

Comment:
Your code is replacing col_name for each row.
It seems to me that the following code would be a little bit more efficient:
```
col_name_parsed = ns.col_name.replace('_', ' ')
result = [row[col_name_parsed] for row in reader]
```